src/io/flutter/run/daemon/DaemonListener.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2016 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.run.daemon;
+
+public interface DaemonListener {
+
+  /**
+   * Process JSON input from the Flutter daemon running on the <code>controller.</code>
```

Comment:
This is intentional. I know looks unusual but it is consistent with the practice of using bold/italic/etc for trailing punctuation.